compiler/qsc_rca/tests/qubits.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+pub mod common;
+
+use common::{check_last_statement_compute_properties, CompilationContext};
+use expect_test::expect;
+
+#[test]
+fn check_rca_for_static_single_qubit_allcation() {
+    let mut compilation_context = CompilationContext::new();
+    compilation_context.update(r#"use q = Qubit();"#);
+    let package_store_compute_properties = compilation_context.get_compute_properties();
+    check_last_statement_compute_properties(
```

Comment:
I'm worried these tests aren't validating what you think they are validating... because the qubit statement gets transformed into an initial call to allocate and a final call to release, the last statemetn in the block will be just part of what was generated from the pass. For example, using [this program in the playground](https://microsoft.github.io/qsharp/?code=H4sIAAAAAAAAEy2OwWrDMBBE7%2FqKAV%2FsS3VPKOTQ0lPAJukHbOwlWYhWirQihJB%2FL1YzxzfwZrz36HCgkK7svPf4YTPRM4pRNl5W1niHLy5zlmQStZHjRQqkgBBEJdAVU4eU4zlTgF3IMJPixKiFF1j8V%2BKepQ1MHea48IdTClwSzYz9Y6qkVsP4tjydA4Ddt1p%2BjFHU%2BqGRmDjT%2BgR7Eu0HbPCrYni2dk0tjBs%2BMdWTWD9sW%2FFyL%2FcHHVFUxvIAAAA%3D) and peeking at the HIR, the last statement of the block will be the call qubit release. So this compute properties is only for the call to `__quantum__rt__qubit_release`. If that's intentional, then it might be worth a comment here to clarify.

Reply:
This is a great catch! I updated the tests to verify the behavior using the variable binding to look at the compute properties of the expression binded to the variable.